src/promptflow-core/promptflow/executor/flow_executor.py

```diff
@@ -880,7 +885,7 @@ def _get_node_referenced_flow_inputs(
     @contextlib.contextmanager
     def _start_flow_span(self, inputs: Mapping[str, Any]):
         otel_tracer = otel_trace.get_tracer("promptflow")
-        with otel_tracer.start_as_current_span(self._flow.name) as span:
+        with start_as_current_span(otel_tracer, self._flow.name) as span:
```

Comment:
Also consider moving the line above `otel_tracer = otel_trace.get_tracer("promptflow")` into this newly created function?

Reply:
will do this in next PR